packages/frontend/src/redux/selectors/crossStateSelectors/selectNEARAsTokenWithMetadata.js

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from 'reselect';
```

Comment:
We keep selectors that belong to a specific slice in the slice file, and these are the selectors that use states from different slices. So it's more `cross slice/reducer` than `cross state`. 

There will be more of them and as we moved `mainSelectors` into one direct file I would prefer to keep them in a separate folder so it will be organized a bit, and not mixed with other files. Is that ok with you?